includes/REST_API/Stories_Controller.php

```diff
@@ -93,6 +105,178 @@ public function prepare_item_for_response( $post, $request ) {
 		return apply_filters( "rest_prepare_{$this->post_type}", $response, $post, $request );
 	}
 
+	/**
+	 * Pre-fills story with demo content.
+	 *
+	 * @return string Pre-filled post content if applicable, or the default content otherwise.
+	 */
+	private function get_demo_content() {
+		$file = WEBSTORIES_PLUGIN_DIR_PATH . 'includes/data/stories/demo.json';
+
+		if ( ! is_readable( $file ) ) {
+			return '';
+		}
+
+		$file_content = file_get_contents( $file ); // phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents, WordPressVIPMinimum.Performance.FetchingRemoteData.FileGetContentsUnknown
+
+		if ( ! $file_content ) {
+			return '';
+		}
+
+		// Make everything localizable.
+
+		// TODO: Replace image URLs, and clean up demo story.
+
+		$kses = new KSES();
+		$kses->init();
+
+		// Page 1.
+
+		$file_content = str_replace(
+			'L10N_PLACEHOLDER_1_1',
+			addslashes(
+				wp_kses(
+					/* translators: demo content used in the "Get Started" story */
+					_x( '<span style="font-weight: 700; color: #fff">Tips </span><span style="font-weight: 100; color: #fff">to make the most of</span>', 'demo content', 'web-stories' ),
+					[
+						'span' => [ 'style' => [] ],
+					]
+				)
+			),
+			$file_content
+		);
+
+		// Page 2.
+
```

Comment:
Would this be cleaner as an array. 

So 
```
   $replacements = [
      'L10N_PLACEHOLDER_2_1' => 
          /* translators: demo content used in the "Get Started" story */
		esc_html_x( 'SET A PAGE BACKGROUND', 'demo content', 'web-stories' ),
     'L10N_PLACEHOLDER_2_2' => 
		/* translators: demo content used in the "Get Started" story */
		esc_html_x( 'Drag your image or video to the edge of the page to set as page background.', 'demo content', 'web-stories' ),   
  ];
 
  foreach( $replacements as $key => $value ) {
        $file_content = str_replace( $key, $value, $file_content );
  }

```

Reply:
Yeah probably 👍 

I could also combine them all in one `str_replace` call too.

But I hesitated to do any optimization here before these strings aren't finished.